pallets/xyk/src/lib.rs

```diff
@@ -331,13 +331,6 @@ pub mod pallet {
 				.checked_add(shares_added)
 				.ok_or(Error::<T>::InvalidLiquidityAmount)?;
 
-			let asset_b_balance = T::Currency::free_balance(asset_b, &who);
-
-			ensure!(
-				asset_b_balance >= amount_b_required,
-				Error::<T>::InsufficientAssetBalance
-			);
-
```

Comment:
Why this is removed ?

Reply:
https://github.com/galacticcouncil/HydraDX-node/blob/12088d88ef7dc3983c984070cd0848ca5f9c1dfc/pallets/xyk/src/lib.rs#L305-L308
https://github.com/galacticcouncil/HydraDX-node/blob/12088d88ef7dc3983c984070cd0848ca5f9c1dfc/pallets/xyk/src/lib.rs#L323-L326 
Previous snippets show that the user's balance is at least `amount_b_required`

Reply:
✔️